### PR TITLE
documentation on generator instances and fork launcher

### PR DESCRIPTION
**Pull request by duylethanh**
This PR contains documentation on:

- generator instances
- fork launcher

### CODE REVIEW

File: docs/launcher.rst [hunk 1]
```diff
@@ -15,7 +15,44 @@ In the following sections, we consider the
...
 ..
-    TODO: Add section on ForkLauncher, Slurm, SGE
+    TODO: Add section on Slurm, SGE
+
+Fork Launcher
+-------------
+
+Simexpal uses the fork launcher if no further configurations are made. The fork launcher launches
+experiments on the local machine. In contrast to the :ref:`QueueLauncher`, the fork launcher
+requires an active connection during the execution of the experiments.
```
Comment (avdgrinten) [inline docs/launcher.rst:25]: It is not clear here what an "active connection" is. Maybe state something like: immediately (in the current terminal) and blocks until completion of the experiments.

Comment (duylethanh) [reply docs/launcher.rst]: Done. I changed it to ``In contrast to the :ref:`QueueLauncher`, the fork launcher launches the experiments immediately in the current terminal and blocks until completion of the experiments``

**Comment by duylethanh** [comment]
PR updated.

**Review by avdgrinten — APPROVED** [review]
LGTM. Shall I merge this now, or do we want to wait for other PRs (to avoid conflicts?).

**Comment by duylethanh** [comment]
Lets wait for now. I will ping here once it can be merged.

**Comment by duylethanh** [comment]
> LGTM. Shall I merge this now, or do we want to wait for other PRs (to avoid conflicts?).

I fixed the merge conflict. PR can be merged now.

**Pull request merged**